app/transport/rest/user_test.go

```diff
@@ -0,0 +1,134 @@
+package rest
+
+import (
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"path"
+	"testing"
+
+	"github.com/go-devs-ua/octagon/app/entities"
+	"github.com/go-devs-ua/octagon/app/globals"
+	"github.com/go-devs-ua/octagon/lgr"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetUserByID(t *testing.T) {
+	logger, err := lgr.New("INFO")
+	if err != nil {
+		t.Fail()
+	}
+
+	tests := map[string]struct {
+		id                   string
+		expectedStatusCode   int
+		expectedResponseBody string
+		getByIDFunc          UserUsecase
+	}{
+		"success": {
+			id:                 "a6703a6b-c054-4ebd-b782-2a96bd4f771a",
+			expectedStatusCode: http.StatusOK,
+			expectedResponseBody: `
+									{
```

Comment:
It's wider than my screen 🤣

Reply:
Sorry sorry, I have small-penis syndrome 😁